Fix - Issue with red flag not updating when clicked

Tompage1994 opened the pull request:
Forcing update of DOM in order to actually allow the changing of the flag on click (setting the store does not cause a DOM reload)

Tompage1994 commented:
/assign @springdo

springdo commented:
@Tompage1994  -  I don't think we need to force the update of the DOM through this handler here; and we probably shouldn't as the `vuex` library should manage the DOM's data lifecycle via the actions and the mutations. The state handler should update it as it does already for the MARK_TODO_COMPLETED flow. I think this issue is simple to fix if you want to go again; but this time just add to the `action.js` file's `addTodo:({})`  method a default value for the important flag and you should be golden, but I could be wrong so have a go!

```javascript
    // debugger
    const todo = {
      title: state.newTodo,
      completed: false,
      important: false
    };
```

Tompage1994 commented:
@springdo fixed that. Hadn't spotted the important field hadn't been added to the new todo item structure. Added now and removed the forced update. Should think you can get mergé with it now
